packages/swingset/src/cli/commands/bootstrap.ts

```diff
@@ -0,0 +1,100 @@
+import fs from 'fs'
+import { error, LOGS } from '../logs'
+import { Bootstrap } from '../types'
+import { detect, PM } from 'detect-package-manager'
+import childProcess from 'child_process'
+
+/*
+This command generates the following file structure, 
+Is supporting this old next versions aka page directory necessary? 
+
+app/
+(swingset)
+  ├ /layout.tsx
+  └ /swingset
+    ├ /page.tsx 
+    └ /[...path]
+      └ /page.tsx
+*/
+
+const bootstrap: Bootstrap = {
+  name: 'bootstrap',
+  description: 'Creates a swingset template in the `app` directory',
+  builder: {},
+  handler: async (_) => {
+    LOGS.bootstrap.start()
+
+    let pkg: PM = 'npm'
+    try {
+      pkg = await detect()
+    } catch (err) {
+      LOGS.bootstrap.unableToInstall()
+      error(err as string)
+      process.exit(1)
```

Comment:
If we can't run `npm|yarn|pnpm install` swingset in their project root, error log why and exit program